raml/raml-information.js

```diff
@@ -73,7 +73,60 @@ var RamlInformation = null;
             return false;
 
         }
-    }
+    };
+
+    /**
+     * This method returns the contentType of the request for the endpoint
+     * @param endpointUri
+     * @returns {null}
+     */
+    RamlInformation.prototype.getContentTypeForPostEndpoint = function (endpointUri) {
+
+        var typeObject;
```

Comment:
getContentType und getTypeObject - zwei verschiedene properties aus der raml. 

Die api (this.api) kapselt die raml schon, alle schritte sind nötig um an das entsprechende property zu kommen.